mobile/status.go

```diff
@@ -1066,6 +1069,32 @@ func writeHeapProfile(dataDir string) string { //nolint: deadcode
 	return makeJSONResponse(err)
 }
 
+// StartMutexProfiling starts mutex profiling and HTTP server for pprof
+func StartMutexProfiling(address string) string {
```

Comment:
At first I thought this should only be part of the `status-backend`. But perhaps having this available from desktop/mobile is a nice to have as well.